src/rgw/rgw_sal_motr.cc

```diff
@@ -457,6 +457,48 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
 
 int MotrUser::remove_user(const DoutPrefixProvider* dpp, optional_yield y)
 {
+  // Remove user info from cache
+  // Delete access keys for user
+  // Delete user info index
+  // Delete user index
+  // Delete email for user - TODO
+  bufferlist bl;
+  // Remove the user info from cache.
+  store->get_user_cache()->remove(dpp, info.user_id.id);
+
+  // Delete all access key of user
+  if (!info.access_keys.empty()) {
+    for(auto acc_key = info.access_keys.begin(); acc_key != info.access_keys.end(); acc_key++) {
+      auto access_key = acc_key->first;
+      int access_del_rc = store->do_idx_op_by_name(RGW_IAM_MOTR_ACCESS_KEY,
```

Comment:
create on variable `rc` and use to hold return code throughout the function. No need to create a variable for it everytime.

Reply:
Done